app/src/main/java/ch/admin/bag/dp3t/networking/models/InfoBoxModel.java

```diff
@@ -16,12 +15,16 @@
 	private String msg;
 	private String url;
 	private String urlTitle;
+	private Boolean isDismissible;
```

Comment:
I think it would be better to use boolean instead of the boxed Boolean, default will be false if not set from the backend.